src/components/LoginPopup/SignupForm.jsx

```diff
@@ -18,9 +18,9 @@ const SignupForm = () => {
 
         try {
             setError("")
-            await signup(data.email, data.password)
-        } catch {
-            return setError("Failed to create account")
+            await signup(data.firstname + ' ' + data.lastname, data.email, data.password)
```

Comment:
Lets change the function parameter to 
const onSumbit = async({password, passwordConfirmation, firstname, lastname, email}) => {
....
}

Reply:
sorry, why?
we don't need the password confirmation in the function, we can check it in the frontend.
I combined the firstname and lastname here so it's cleaner later on.

Reply:
wait, you commented on the wrong line, I thought you were referring to something else.
not a big difference but sure I can do that 👌🏼